src/litdata/streaming/serializers.py

```diff
@@ -387,13 +388,37 @@ def can_serialize(self, data: float) -> bool:
         return isinstance(data, float)
 
 
+class TIFFSerializer(Serializer):
```

Comment:
> Many compression and predictor schemes are supported via the imagecodecs library, including LZW, PackBits, Deflate, PIXTIFF, LZMA, LERC, Zstd, JPEG (8 and 12-bit, lossless), JPEG 2000, JPEG XR, JPEG XL, WebP, PNG, EER, Jetraw, 24-bit floating-point, and horizontal differencing.

From the tifffile, it seems it comes with compression built-in. It might be worth to see if there is a compressed version of the file to use

Reply:
Also, this is the kind of serializer you want to customize and pass to the `optimize` and `StreamingDataset` to be max performance from it.

Reply:
Would be interesting to explore that as an option - however we generally don't want to compress remote sensing imagery.

Reply:
From checking the docs, users are free to apply any compression while generating tiff files using imwrite. So I guess it is fine.